helm/hpcc/templates/service-account.yaml

```diff
@@ -9,16 +9,12 @@ kind: Role
 metadata:
   name: hpcc
 rules:
-  - apiGroups:
-      - "batch"
-    resources:
-      - jobs
-    verbs:
-      - get
-      - create
-      - list
-      - delete
-      - watch
+  - apiGroups: [ "" ] # core API group
+    resources: [ "pods" ]
+    verbs: [ "get", "create", "list", "delete", "watch" ]
```

Comment:
Just get is probably all that is needed (and better to minimize such things)

Reply:
with just "get" I get:
```
0000000F PRG 2020-06-16 12:43:06.901     1    29 "kubectl get schedule status error: Error from server (Forbidden): pods is forbidden: User "system:serviceaccount:default:hpcc" cannot list resource "pods" in API group "" in the namespace "default"
"
```

But if I make it just "list" it works.

We're issuing kubectl get pods --selector=...

it seems that's translates into a "list" API request.

Perhaps best to keep both "get" and "list" in?

Reply:
Ok